package.json

```diff
@@ -32,7 +32,7 @@
     "uglifyjs-webpack-plugin": "^2.2.0"
   },
   "devDependencies": {
-    "nodemon": "^1.19.2",
```

Comment:
Was that intended?

Reply:
yeah well it was necessary anyway to update that one so